tests/integration/integration.go

```diff
@@ -205,17 +205,13 @@ func K3sTestLock() (int, error) {
 // K3sStartServer acquires an exclusive lock on a temporary file, then launches a k3s cluster
 // with the provided arguments. Subsequent/parallel calls to this function will block until
 // the original lock is cleared using K3sKillServer
-func K3sStartServer(inputArgs ...string) (*K3sServer, error) {
+func K3sStartServer(inputArgs []string) (*K3sServer, error) {
```

Comment:
Why did you refactor `K3sStartServer` to take a list instead of being a variadic function? It doesn't seem to be necessary to support the test you're adding, and contributes a lot of other churn to the PR without any clear reason as to why.

Reply:
`K3sCmd` and others are still variadic and I don't see any commentary as to why they shouldn't be.